package.json

```diff
@@ -10,17 +10,18 @@
     "test": "jest --watchAll"
   },
   "dependencies": {
+    "@hookform/resolvers": "^2.9.11",
     "eslint": "8.35.0",
     "eslint-config-next": "13.2.3",
     "next": "13.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.43.5",
     "react-icons": "^4.8.0",
-    "react-native-linear-gradient": "^2.6.2",
-    "react-use": "^17.4.0",
     "styled-components": "^5.3.8",
-    "uuid": "^9.0.0"
+    "uid": "^2.0.1",
+    "use-local-storage": "^2.3.6",
```

Comment:
The package `use-local-storage` isn't used within your project and I wonder if you even need it. `uid` hasn't been used either.

Reply:
yes you're right, I installed and used the packages before I used the form-hook, I installed use-local-storage accidentally, i fix that